app/Controller/ProfileController.php

```diff
@@ -426,7 +426,7 @@ public function edit() {
             }
             $this->set('user', $user);
         }
-        $this->set('sexes', array('male'=>__('male'),'female'=>__('female')));
+        $this->set('sexes', array('male'=>__('male'),'female'=>__('female'),'other'=>('other'),'na'=>('prefer not to say')));
```

Comment:
I suspect that adding this will fix your missing translations issue.
```suggestion
        $this->set('sexes', array('male'=>__('male'),'female'=>__('female'),'other'=>__('other'),'na'=>__('prefer not to say')));
```

Reply:
..oops! Works like a charm.